html/modules/custom/gho_figures/src/Form/GhoFiguresImportNeedsAndRequirementsFiguresForm.php

```diff
@@ -1175,7 +1175,7 @@ public static function checkMandatoryField($name, array $definition, array $data
   public static function call(array $arguments, &$data) {
     $callable = array_shift($arguments);
     $arguments[] = &$data;
-    return call_user_func_array($callable, $arguments);
+    return call_user_func_array($callable, array_values($arguments));
```

Comment:
@orakili this was a PHP8 breaking change that we had to address. If any other custom code in various projects is using this function with named arguments, it needs to be upgraded.

Reply:
Or change https://github.com/UN-OCHA/gho-2022-site/blob/f62ad67018094ca96533384a1d08c3b42f76b9b8/html/modules/custom/gho_figures/src/Form/GhoFiguresImportNeedsAndRequirementsFiguresForm.php#L676 and remove the named argument `['name' => $name]`